<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchGroupBox</class>
 <widget class="QGroupBox" name="SearchGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Szukanie plików</string>
  </property>
  <property name="title">
   <string/>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Zapytanie</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="anyRadio">
          <property name="text">
           <string>którekolwiek</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="allRadio">
          <property name="text">
           <string>wszystkie</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="exactRadio">
          <property name="text">
           <string>dokładna fraza</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>szukaj:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="szukajLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>pliki:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="allfilesRadio">
          <property name="text">
           <string>wszystkie</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="videoCheckBox">
          <property name="text">
           <string>video</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="audioCheckBox">
          <property name="text">
           <string>audio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="textCheckBox">
          <property name="text">
           <string>text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fotoCheckBox">
          <property name="text">
           <string>foto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="dirCheckBox">
          <property name="text">
           <string>katalogi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="customCheckBox">
          <property name="text">
           <string>typu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="extLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="txtCheckBox">
          <property name="text">
           <string>Szukaj w tekstach i tytułach newsów i programów</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="SzukajButton">
        <property name="toolTip">
         <string>Esc przerywa szukanie</string>
        </property>
        <property name="text">
         <string>Szukaj</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Wyniki</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
